modular_citadel/code/modules/mob/cit_emotes.dm

```diff
@@ -207,7 +205,7 @@
 	emote_type = EMOTE_AUDIBLE
 
 /datum/emote/living/merp/run_emote(mob/living/user, params)
-	if(ishuman(user))
+	if(iscarbon(user) || ispAI(user))
```

Comment:
Despite the fact `mob_type_allowed_typecache` existed for a reason, you had to do this instead.